src/herder/HerderSCPDriver.cpp

```diff
@@ -897,6 +928,7 @@ HerderSCPDriver::recordSCPExecutionMetrics(uint64_t slotIndex)
     }
 
     auto& SCPTiming = SCPTimingIt->second;
+    SCPTiming.mSCPProgressState = SCPProgressState::EXTERNALIZE;
```

Comment:
by moving to "per ledger", you won't need to keep track of EXTERNALIZE anymore as this function will be the only place where you interact with metrics